classes/Link.php

```diff
@@ -1377,10 +1395,27 @@ public function getBaseLink($idShop = null, $ssl = null, $relativeProtocol = fal
             $shop = Context::getContext()->shop;
         }
 
+        $domainSSL = $shop->domain_ssl;
```

Comment:
Looks like this a duplicate from the previous code. Can you please put it in a separate function for easier maintenance ?

Reply:
Hello,
Yes, without any problem I do that.